cmd/client_test.go

```diff
@@ -89,14 +94,18 @@ func TestClient_CreateJobSpec(t *testing.T) {
 		nJobs   int
 		errored bool
 	}{
-		{"bad input", 0, true},
+		{"{bad json}", 0, true},
+		{"bad/filepath/", 0, true},
 		{`{"initiators":[{"type":"web"}],"tasks":[{"type":"NoOp"}]}`, 1, false},
 		{`{"initiators":[{"type":"runAt","time":"2018-01-08T18:12:01.103Z"}],"tasks":[{"type":"NoOp"}]}`, 2, false},
+		{"../internal/fixtures/web/end_at_job.json", 3, false},
+		{expandGoPath("internal/fixtures/web/hello_world_job.json"), 4, false},
+		{expandGoPath("internal/fixtures/web/invalid_cron.json"), 4, true},
```

Comment:
I understand we might want to test the `~/some_job.json` but for now lets skip that scenario. We already have coverage and that edge case is handled by `homedir`, no need to test a third party package.